addons/mail_group/models/mail_group.py

```diff
@@ -589,15 +589,15 @@ def _send_subscribe_confirmation_email(self, email):
         template = self.env.ref('mail_group.mail_template_list_subscribe')
         template.with_context(token_url=confirm_action_url).send_mail(
             self.id,
-            force_send=True,
```

Comment:
Little strange that you moved the force_send since it actually respected the method parameters order.

EDIT: oh wait.. you are probably ordering them alphabetically, you madman...